docs/tutorials/basics.md

```diff
@@ -4,39 +4,82 @@ sidebar_label: Getting Started
 featured: true
 ---
 
-## Become a coversational expert with YM two minutes at a time
+Explore video-based learning to effortlessly enhance your skills! This section is tailored for busy learners seeking concise and engaging experiences. These videos provide visual demonstrations, step-by-step processes, and walkthroughs of various products of Yellow.ai in a more engaging, accessible, and efficient manner. Elevate your learning journey today 🏃🏻‍♂️!
 
-This section is for all the learners who LOVE to LEARN on the GO 🏃🏻‍♂️ Time is precious and not everyone likes to take the long trail of step by step learning but get quick answers to tricky questions. This section is for the same kind of learners.
 
 :::info
-For more information about yellow.ai and learning, please visit this link - [Ascend](https://ascend.yellow.ai/) and sign up here to access all the resources bot building courses, virtual classroom sessions, video/audio content and much more.
+To access all the resources, including bot building courses, virtual classroom sessions, video/audio content, and more, visit [this link](https://ascend.yellow.ai/) and sign up to Yellow.ai's Ascend.
 :::
 
 
-### How to Sign up to Yellow.ai and create your first bot? 
-This video will take you through the steps of signing up on https://cloud.yellow.ai/
+### Sign up to Yellow.ai and create your first bot
+
+Join Yellow.ai to unlock the power of building and customizing your own bot. With an easy sign-up process, you'll be able to dive right in and start creating your bot from scratch. Get started today and experience the excitement of bringing your ideas to life with Yellow.ai's user-friendly bot creation platform.
+
+
+<p align="center"><iframe width="560" height="315" src="https://www.youtube.com/embed/5RhnA_jOdlM" title="YouTube video player" frameborder="0" allow="autoplay; clipboard-write; picture-in-picture" allowfullscreen="True"></iframe></p>
 
-:::info
-Don't forget to verify your email in your inbox after sign up.
-:::
 
-<p align="center"><iframe width="560" height="315" src="https://youtu.be/5RhnA_jOdlM" title="YouTube video player" frameborder="0" allow="autoplay; clipboard-write; picture-in-picture" allowfullscreen="True"></iframe></p>
 
 ### Platform overview
 
-<p align="center"><iframe width="560" height="315" src="https://youtu.be/QYRdym4wZf8" title="YouTube video player" frameborder="0" allow="autoplay; clipboard-write; picture-in-picture" allowfullscreen></iframe></p>
+Welcome to our platform overview video! In this brief introduction, we will take you on a tour of our powerful platform, showcasing its key features and functionalities. Discover how our platform empowers you to create, customize, and deploy intelligent bots with ease. From seamless integrations to advanced analytics, we have everything you need to streamline your conversational AI initiatives. Let's dive in and explore the possibilities that await you on our platform.
```

Comment:
just another suggestion, we can include some of these in the intro and keep this description simple as all the descriptions sound like a doc intro.